app/helpers/comments_helper.rb

```diff
@@ -291,4 +291,37 @@ def fallback_url_for_comment(comment, options = {})
 
   end
 
+  # find the parent of the commentable
```

Comment:
@sarken Not sure this is necessary? The 'ultimate_parent' method should return the same results - the work for a comment on a chapter or another comment, and the admin post or tag for a comment on one of those.